config/schema/og.schema.yml

```diff
@@ -1,5 +1,5 @@
 og.settings:
-  type: config_object
+  type: config_entity
```

Comment:
Not sure why this was changed? This `og.settings` schema is just for simple config, not a config_entity. Is this not why the tests are breaking?